README.md

```diff
@@ -98,12 +98,22 @@ have set the `NODE_ENV` variable.
 
 ### Convert Preferences Data
 
-GPII has 2 set of preferences JSON5 data files:
-
-* The preferences files for running GPII are located at %gpii-universal/testData/preferences
-* The preferences files for running node tests are located at %gpii-universal/tests/data/preferences
-
-When any preferences file in either one of these 2 directories are modified, running `npm run postinstall` will generate
+GPII has two sets of preferences JSON5 data files, located at `%gpii-universal/testData/preferences` and `%gpii-universal/tests/data/preferences`:
+
+* The preferences files for running GPII and for integration tests are located at
+  `%gpii-universal/testData/preferences`.  These files are converted into two types of preferences safes and GPII keys:
+  * They are converted into `snapset` preferences safes and GPII keys and placed into the
+    `%gpii-universal/build/dbData/snapset/` folder.   These are used to update the snapsets in CouchDB when GPII is
```

Comment:
Reading "when GPII is run in a production or staging configuration" reminds me the use of config files in `gpii/configs` directory. All those configs use pouchDB at the backend.

Probably adjust this sentence to express datasets in this folder are:
1. loaded into the production and staging CouchDB in the real clouds.
2. loaded into the pouchDB when GPII runs locally (regardless which config is used.).

Reply:
Thanks @cindyli I've reworked the whole section to:
1. better sync up with the `testData\dbData\README.txt`, and
2. adjust the wording to reflect your suggestions.